lib/gc/ExecutionEngine/CPURuntime/MemoryPool.cpp

```diff
@@ -239,16 +239,24 @@ static thread_local FILOMemoryPool mainMemoryPool_{mainChunkSize};
 // if the current thread is a worker thread, use this pool
 static thread_local FILOMemoryPool threadMemoryPool_{threadlocalChunkSize};
 
-extern "C" void *gcAlignedMalloc(size_t sz) noexcept {
+#if defined _WIN32 || defined __CYGWIN__
```

Comment:
Maybe we should move this macro to a shared header in `CPURuntime` folder, as many cpp files will need it?

Reply:
Sure.

Reply:
May be move it even higher? I believe it's required not only for CPURuntime. What about renaming gc_version.h to gc.h and putting the common staff there?

Reply:
> May be move it even higher? I believe it's required not only for CPURuntime.

What about to do this only when there is a real requirement?

> renaming gc_version.h to gc.h and putting the common staff there

The name `gc_version.h` conveys more specific information and may be a better name. The newly added `CPURuntime/Utils.h` and `gc_version.h` contains macros with totally different purpose. Currently these is no file which needs to include them both. So putting them together may not be necessary.

Reply:
> What about to do this only when there is a real requirement?

Currently, except of CPURuntime, we need it at least for lib/gc/ExecutionEngine/OpenCLRuntime/OpenCLRuntimeWrappers.cpp and src/dnnl/dnnl_graph_compiler.cpp